NBXplorer/Controllers/MainController.cs

```diff
@@ -854,13 +854,14 @@ async Task<(uint256 BlockId, Transaction Transaction, DateTimeOffset BlockTime)>
 				return getTransactionsResult;
 
 			var network = this.GetNetwork(cryptoCode, false);
+			var immature = CalculateBalance(network, transactions.ImmatureTransactions);
 			var balance = new GetBalanceResponse()
 			{
 				Confirmed = CalculateBalance(network, transactions.ConfirmedTransactions),
 				Unconfirmed = CalculateBalance(network, transactions.UnconfirmedTransactions),
-				Immature = CalculateBalance(network,transactions.ImmatureTransactions),
 			};
-			balance.Total = balance.Confirmed.Add(balance.Unconfirmed).Add(balance.Immature);
+			balance.Available = balance.Confirmed.Add(balance.Unconfirmed);
+			balance.Total = balance.Available.Add(immature);
```

Comment:
I think this may lead to double lead to double counting of immature txns.

Reply:
Yes, I rewrote the test to use hard coded number to make sure it was calculating right.

I also put back the field `immature`. It is clearer with it.